base_layer/core/src/validation/block_validators.rs

```diff
@@ -149,6 +144,60 @@ impl<B: BlockchainBackend> PostOrphanBodyValidation<B> for BodyOnlyValidator {
     }
 }
 
+/// This is a test only validator as it skips the mined height rule of a input.
```

Comment:
if this is a test only validator, it should be in a` #[cfg(test)]` block and in it's own file. Otherwise it may get used